manager/state/raft/raft.go

```diff
@@ -957,7 +966,7 @@ func (n *Node) SubscribePeers() (q chan events.Event, cancel func()) {
 func (n *Node) GetMemberlist() map[uint64]*api.RaftMember {
 	memberlist := make(map[uint64]*api.RaftMember)
 	members := n.cluster.Members()
-	leaderID := n.Leader()
+	leaderID, _ := n.Leader()
```

Comment:
How about adding `if err != nil { leaderID = 0 }`. I know it's not necessary but it means we don't depend too much on the behavior of `n.Leader`.